### PR TITLE
More fixes for bugs introduced due to PR #462

### PR DESCRIPTION
**Pull request by vijayaditya**
the scripts now run till get_egs.sh stage. Will update if there are issues.

### CODE REVIEW

File: egs/swbd/s5c/local/chain/run_tdnn_4q.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@ stage=10
 train_stage=-10
 get_egs_stage=-10
 speed_perturb=true
-dir=exp/chain/tdnn_4q  # Note: _sp will get added to this if $speed_perturb == true.
+dir=exp/chain/tdnn_4q_trial2  # Note: _sp will get added to this if $speed_perturb == true.
```
Comment (danpovey) [inline egs/swbd/s5c/local/chain/run_tdnn_4q.sh]: I'm not sure if you really meant this but I'll merge anyway for now.

Comment (danpovey) [inline egs/swbd/s5c/local/chain/run_tdnn_4q.sh]: actually, i'l lwait till you fix this.

Comment (vijayaditya) [inline egs/swbd/s5c/local/chain/run_tdnn_4q.sh]: I would suggest waiting till few iterations are done. 
I had copied this recipe from a branch with a lot of other changes, which is the reason for these errors.

**Comment by vijayaditya** [comment]
Training proceeds without errors. Good to merge.

**Pull request merged**